src/pages/extension-manager/extension-developed-by-adobe/ue-product-picker/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: AEM Product Picker for Universal Editor
+description: Bring Adobe Commerce product data into your project effortlessly with the UE extension, allowing you to manage and utilize this data directly within editor.
+contributors:
+- AdobeDocs/uix
+---
+
+# AEM Product Picker for Universal Editor
+
+This extension for [Universal Editor](https://experienceleague.adobe.com/en/docs/experience-manager-cloud-service/content/implementing/developing/universal-editor/introduction) integrates [Adobe Commerce](https://business.adobe.com/products/magento/magento-commerce.html) data into your project, allowing users to select or remove product data directly within the editor.
+
+Functionality Overview:
+
+- [Replaces the standard UI](https://developer.adobe.com/uix/docs/services/aem-universal-editor/api/item-types-renderers/) for any field type component in the properties rail with a new component UI.
+- The new component features a list of selected products and includes a button to open the catalog for adding new products or removing selected ones.
+- Clicking the button lets users navigate through categories, select products from any category, or use search functionality to find specific products.
+- Users can sort selected products using drag and drop within the properties rail.
+- Utilizes components from [Adobe React Spectrum](https://react-spectrum.adobe.com/react-spectrum/index.html).
+
+![Overview - Right Panel](pp-overview-right-panel.png)
+
+![Overview - Modal](pp-overview-modal.png)
+
+## Configuration via Extension Manager
```

Comment:
It will take me a long time to explain this in a comment, but the current structure is more dependent on client requests. There was one request—so a section appeared; then a second request—a second section; a third—a third section. Everyone focuses on the specific case they need, and many don't read the documentation fully (is it bad or good can't say).

Initially, it was in a general format, as we adhere to on other pages... but now it has been tailored to customer requests. There aren't many of them, and I would be happy to adjust the structure. However, right now, everyone is more concerned with the accessibility of the functionality. With the current documentation, 2-3 parameters are enough to set everything up; the main thing is to provide access to ext.

I would like to present the documentation as it is and improve it iteratively if it gains popularity.